azure-pipelines/pull-request-validation/common4j.yml

```diff
@@ -17,6 +21,8 @@ pool:
 jobs:
 - job: build_test
   displayName: Build & Test
+  env:
+    MOCK_API_URL: https://native-auth-mock-api.azurewebsites.net/
```

Comment:
Why is this here? Shouldn't this PR be introducing the integration with the environment var, so we can test that?